feat: improvements on docs (#24)
waku-org/waku.guide: docs/Concepts/5.md

---
title: Transports in Waku
---

Waku v2 is built in top of libp2p, and like libp2p it strives to be transport agnostic. We define a set of recommended transports in order to achieve a baseline of interoperability between clients.

This section describes these recommended transports:

> - Waku nodes uses TCP to communicate by default. A service node should be using TCP to connect to other nodes and listen to

Reviewer: ```suggestion
- Waku nodes uses TCP to communicate by default. A service node should be using TCP for communicate with other nodes.
```

Reviewer: A section of state of NAT would be good here so reader know what to expect. cc @richard-ramos @jm-clius

> - In environments where TCP is not available, such as browser, secure websocket is used. service nodes are encouraged to setup a SSL certificate to enable incoming connections from browser and serve them.

Reviewer: I'd say _browser -> websocket_ instead of _if not tcp then websocket_
